bin/worker.php

```diff
@@ -41,14 +41,14 @@
 
 Addon::loadHooks();
 
-$spawn = (($_SERVER["argc"] == 2) && ($_SERVER["argv"][1] == "spawn"));
+$spawn = Argument::get('spawn',$_SERVER["argc"], false, 'boolean');
 
 if ($spawn) {
 	Worker::spawnWorker();
 	killme();
 }
 
-$run_cron = (($_SERVER["argc"] <= 1) || ($_SERVER["argv"][1] != "no_cron"));
+$run_cron = Argument::get('cron',$_SERVER["argc"], false, 'boolean');
```

Comment:
`$_SERVER["argc"]` holds the argument count, not the argument array, please use `$_SERVER['argv']` instead.
Additionally, space is missing after a comma.